proxy/coco/src/project.rs

```diff
@@ -12,6 +13,18 @@ pub use create::{Create, Repo};
 pub mod checkout;
 pub use checkout::Checkout;
 
+fn set_include_path<P>(
+    repo: &git2::Repository,
+    include: &include::Include<P>,
+) -> Result<(), git2::Error>
+where
+    P: AsRef<std::path::Path>,
+{
+    let mut config = repo.config()?;
+
+    config.set_str("include.path", &include.file_path().display().to_string())
```

Comment:
@FintanH Can we get rid of the magic string here? Maybe `librad;:git::include` could export a const?

Reply:
You didn't add this after all the forrays into link? :scream:

Reply:
:facepalm: forgot my own todo...PR forthcoming

Reply:
radicle-dev/radicle-link#335